actions/ci-beholder-validator/README.md

```diff
@@ -0,0 +1,164 @@
+# Schema Evolution Validator
```

Comment:
This is not accurate and will be changed once the testing can be completed satisfactorily